test/controllers/ContentTest.scala

```diff
@@ -710,7 +708,7 @@ class ContentTest extends ControllerBaseTest {
     }
 
     "display form error message (transferor email has consequent dots)" in {
-      val request = FakeRequest().withFormUrlEncodedBody("transferor-email" -> "ex..ample@example.com")
+      val request = FakeRequest().withFormUrlEncodedBody("transferor-email" -> "ex...ample@example.com")
```

Comment:
This test already has this error here. If the former test is now passing we should probably take a look at this